terramate.tm.hcl

```diff
@@ -0,0 +1,24 @@
+stack {
+  name        = "terraform-module-template"
```

Comment:
just noticed this. Remember to update the stack `name` and the `description` just below after copying from the template :smile_cat: